javascript/node/selenium-webdriver/chromium.js

```diff
@@ -720,12 +721,20 @@ class Driver extends webdriver.WebDriver {
    * @param {string} debuggerAddress
    * @return {string} Returns parsed webSocketDebuggerUrl obtained from the http request
    */
-  async getWsUrl(debuggerAddress) {
-    let request = new http.Request('GET', '/json/version')
+  async getWsUrl(debuggerAddress, target) {
```

Comment:
Hi @raju249 ,

can we check for "target" here and handle the error if user is not providing a correct target, like the supported are `browser` and `page` for as i know, because if user doesn't provide any it may throw `target.toLowerCase' of undefined.

Also adding a JSDOC comment will be helpful what parameters actually goes in there for the user

Reply:
Oh yes, fixed it. Please take a look 🙇